### PR TITLE
Show/hide background image smoothly with CSS transition

### PR DESCRIPTION
**Pull request by mikelei8291**

### CODE REVIEW

File: ursine-umbra.scss [hunk 1]
```diff
@@ -1119,6 +1119,9 @@ footer.ty-footer {
     -moz-background-size: cover;
     -o-background-size: cover;
     opacity: 0;
+    -webkit-transition: opacity 0.2s ease-out;
+    -o-transition: opacity 0.2s ease-out;
```
Comment (krokofant) [inline ursine-umbra.scss]: Probably don't need the vendor prefixes

Comment (mikelei8291) [reply ursine-umbra.scss]: Removed, but only for my previous commit. Should you consider open another issue or pr for removing other vendor prefixed rules for the original branch?

**Comment by aCluelessDanny** [comment]
Ah, unfortunately this is something I actually tried in the past, but it led to a problem where the first line of text became either difficult to select or unselectable (#13). I tried this again, and it still happens. This is because of how I implemented the splashscreen, & there's not much around it because I haven't found another way to style `#write` when there's no content.
I recently opened an issue on Typora to add something small just for that (typora/typora-issues#3007), but for now, I guess we'll have to wait it out

**Comment by mikelei8291** [comment]
Alright, let's see if they can implement this small feature.

**Pull request closed without merging**